package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ion-range-slider-angularjs",
+  "version": "1.0.0",
```

Comment:
```
   "version": "1.1.1",
```

This is what is currently defined inside bower.json. Can you change this version number ?